projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -129,6 +157,10 @@ This discrepancy can indicate potential issues in the ontology's design, as equi
 
 8. Analyzing Class Hierarchies for Inconsistent Type Definitions
 
+Title: Inconsistent Class Type Definition
+Constraint Description: Detects classes assigned multiple, different types.
+Severity: Error
+
```

Comment:
this is detecting classes which are not declared as class, I am not sure whether this is what you want to do

Reply:
I have to change this one